packages/odyssey-react/src/components/Form/Form.tsx

```diff
@@ -64,27 +65,39 @@ export const Form = withTheme(
     const omitProps = useOmit(rest);
 
     return (
-      <form {...omitProps} className={styles.root} ref={ref}>
-        <header className={styles.header}>
+      <Box as="form" {...omitProps} className={styles.root} ref={ref}>
+        <Box as="header" className={styles.header}>
```

Comment:
why two nested Box here? I'm not sure I follow why we can't just used `header` as the child here? Is there some kind of user agent style that is problematic and requires all the base styles from Box to render correctly?

Reply:
bumping this question again cc @arnoldsandoval-okta

Reply:
Missed this. fixing now.